docs/content/guides/getting-started/setting-options.md

```diff
@@ -137,12 +196,10 @@ const hot = new Handsontable(container, {
 hot.getCellMeta(0, 0).readOnly;
 ```
 :::
-:::
```

Comment:
Removing this breaks the page layout, resulting in some of the examples not being displayed etc.